src/main/java/model/item/Item.java

```diff
@@ -15,27 +15,38 @@
  */
 public class Item {
 
-    protected String name = "";
-    protected String typ = "";
-    protected String subTyp = "";
-    protected String requirement = "";
-    protected String effect = "";
-    protected Rarity rarity = Rarity.common;
-    protected Currency currency = new Currency(0);
-    protected int tier = 1;
-    protected FloatProperty amount = new SimpleFloatProperty(1);
+    protected String name;
+    protected String type;
+    protected String subtype;
+    protected String requirement;
+    protected String effect;
+    protected Rarity rarity;
+    protected Currency currency;
+    protected int tier;
```

Comment:
should Tier be an enum?

Reply:
Strictly seen can tier be any arbitrary number higher than 0.
It's only limited in my use-case to 5 (and that is not always the case).